src/gz/bufread.rs

```diff
@@ -270,7 +270,8 @@ impl<R: BufRead + Write> Write for GzEncoder<R> {
     }
 }
 
-/// A gzip streaming decoder
+/// Decode a single member of a gzip file. Prefer [MultiGzDecoder] for
+/// most uses.
```

Comment:
Since the struct itself does NOT decode, an expression like "A decoder..." is appropriate.

Reply:
Fixed!